cpp/examples/minimal_build/Dockerfile

```diff
@@ -22,5 +22,6 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update -y -q && \
     apt-get install -y -q --no-install-recommends \
       build-essential \
-      cmake && \
+      cmake \
+      ninja-build && \
```

Comment:
Is this required? If not strictly required, maybe comment that this is an optimization, not a hard dependency (since this is the minimal dockerfile)

Reply:
Yes